src/components/Navbar.jsx

```diff
@@ -37,10 +48,10 @@ const NavBar = () => {
           <Typography variant="h6" className={classes.title}>
               JSONE
           </Typography>
-          <Button color="inherit">Home</Button>
-          <Button color="inherit">Interfaces</Button>
-          <Button color="inherit">About</Button>
-          <Button color="inherit">Playground</Button>
+          <Link className={classes.navLink} to="/">Home</Link>
+          <NavLink smooth className={classes.navLink} to="/#interface">Interface</NavLink>
+          <Link className={classes.navLink} to="/about">About</Link>
+          <Link className={classes.navLink} to="/playground">Playground</Link>
```

Comment:
Shouldn't these all be `NavLink`?  And, I see that it's linking to `/about` instead of the hash version, `#about`.

Reply:
Linking to /about is intentional. 
I've used the hashed link for **#interface** because that is a single page routing. We need not switch between pages but to jump on a section of home page i-e **interface**. Whereas **about** is not a section of **home page** to jump on but an individual page on route **/about**. 
I would use **/#about** if it was on the main page under or above the interface section.

Reply:
Ah, got it.  Maybe later we could move interface to its own page rather than the home page, but for now this totally makes sense.

Reply:
@djmitche sounds good.